README.md

```diff
@@ -270,8 +277,44 @@ It is perfectly fine to chain multiple `views`, `props` calls etc in arbitrary o
 
 It is also possible to define lifecycle hooks in the _actions_ object, these are actions with a predefined name that are run at a specific moment. See [Lifecycle hooks](#lifecycle-hooks-for-typesmodel).
 
-_Tip: `(self) => ({ action1() { }, action2() { }})` is ES6 syntax for `function (self) { return { action1: function() { }, action2: function() { } }}`, in other words; it's short way of directly returning an object literal.
-For that reason a comma between each member of a model is mandatory, unlike classes which are syntactically a totally different concept._
+TypeScript:
```

Comment:
Could you move the typescript not bit down, to the typescript section below in the README (or institute a separate page for TS in the docs/ folder and link to it). I fear having the TS explanation on the type is quite overwhelming and not relevant for everybody